index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="robots" content="noindex">
     <meta name="title" content="WorkAdventure Starter Kit">
-    <title>WorkAdventure test map</title>
+    <title>WorkAdventure test map upload</title>
```

Comment:
This page is not meant to test the upload of the map, but just to test the map itself. So can you explain this change?